### ISSUE TITLE
example.py: NameError: name 'data' is not defined (solved)

### ISSUE THREAD

**Issue by Gajeski** [rileypsmith/sklearn-som#5]
I copy pasted the code from your example and the following exception was thrown: "NameError: name 'data' is not defined" at the .ft() method in som.py. Changing the argument name 'X'  to 'data'  in the function header (.fit() in som.py) seems to fix the issue.

**Comment by omargalal20084** [comment]
I still have the same issue

**Comment by rileypsmith** [comment]
Hi @spiderman20084, what version of sklearn-som are you using? There was a bug with a variable having the wrong name in the function signature, but it should be patched in version 1.0.1. Are you still seeing this error in sklearn-som 1.0.1?

**Comment by gokerDEV** [comment]
I have the same issue too

```
pip show sklearn_som    
Name: sklearn-som
Version: 1.0.1
Summary: A simple planar self organizing map
Home-page: https://github.com/rileypsmith/sklearn-som
Author: Riley Smith
```

Still, I have the "data" variable instead of "X" in the code.
`n_samples = data.shape[0]`

**Comment by rileypsmith** [comment]
@gokercebeci thank you for this information. I am able to reproduce the issue with a clean install of sklearn-som v. 1.0.1. I think the Github project is ahead of the PyPi distribution. I will issue a fix as soon as I get home this afternoon. 

Thanks for bringing this back to my attention.

**Comment by rileypsmith** [comment]
I have patched the bug and released a new version of sklearn-som, version `1.1.0`. Please upgrade to version `1.1.0` and let me know if the issue persists. Note that you can no longer use `from som.som import SOM` and must instead use `from sklearn_som.som import SOM`.

Marking this as closed for now but please re-open if there is still an issue.
